lib/player.js

```diff
@@ -7423,7 +7423,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         // If we are in Live, and we are very close to the live edge with a rate
         // between 0 and 1, it is not necessary to cancel since we are moving
         // away from the edge.
```

Comment:
Please adjust the comment so it's clear we do not cancel trick play on VOD.

Reply:
Done!